src/components/hamburger/Sidebar.tsx

```diff
@@ -1,21 +1,36 @@
+import React, { useState } from 'react';
 import { AnimatePresence, motion } from 'framer-motion';
 
 import fbIcon from '../../assets/icons/hamburger/facebook-logo.svg';
 import igIcon from '../../assets/icons/hamburger/instagram-logo.svg';
 
+import './styles/index.css';
+
 type SidebarProps = {
   isOpen: boolean;
 };
 
 export const Sidebar = ({ isOpen }: SidebarProps) => {
-  const variants = {
-    open: { opacity: 1, x: 0 },
-    closed: { opacity: 0, x: '100%' },
-  };
+  const [activeNavItem, setActiveNavItem] = useState('');
 
-  const handleLinkClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
+  const handleLinkClick = (
```

Comment:
here, we don't have to use events at all. we could directly pass the `href` value into this function. Basically, we can remove this function and just set the active item directly on click